tests/drawing/test_draw.py

```diff
@@ -183,3 +183,41 @@ def test_draw_hypergraph_hull(edgelist8):
     assert ax.collections[0].get_zorder() == 4  # nodes
 
     plt.close()
+
+
+def test_draw_multilayer(edgelist8):
```

Comment:
Seeing that this is mostly testing the number of things drawn (right?). If so, I'd suggest a more descriptive test name. Test function names can be super long, it doesn't matter.